install.sh

```diff
@@ -394,6 +417,11 @@ os_is_deb ()
     ( which apt-get && which dpkg-query ) 2>/dev/null 1>&2
 }
 
+os_is_dar ()
+{
+    ( which brew  ) 2>/dev/null 1>&2
```

Comment:
This check isn't correct as homebrew can also [exist on Linux](https://docs.brew.sh/Homebrew-on-Linux).

Reply:
We currently check for OSX like [this](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/a219cf1ef07a33324bbb450314c7a6172634eab6/install.sh#L9).

Reply:
```
if [[ "$OSTYPE" == "darwin"* ]]; then
```

Reply:
@jameshilliard Does the suggestion above make sense to you? This seems to be better to me.

Reply:
I think `uname` is also perfectly good for this.

Reply:
Copy paste a portion or all of the shell script into a formatter for a 2nd opinion, e.g.: https://www.shellcheck.net/